tests/v2/validation/provisioning/rke1/provisioning_node_driver_test.go

```diff
@@ -190,10 +190,10 @@ func (r *RKE1NodeDriverProvisioningTestSuite) TestProvisioningRKE1ClusterDynamic
 		psact  string
 	}{
 		{provisioning.AdminClientName.String(), r.client, r.psact},
-		{provisioning.StandardClientName.String(), r.standardUserClient, r.psact},
+		// {provisioning.StandardClientName.String(), r.standardUserClient, r.psact},
```

Comment:
This seems like a typo? Or was this meant to be commented out?

Reply:
this is a typo, I'll fix it.

Reply:
fixed